Dockerfile

```diff
@@ -139,7 +139,7 @@ ARG FIN_TOKEN=
 RUN git clone https://$FIN_TOKEN:x-oauth-basic@github.com/ROCmSoftwarePlatform/Fin.git $FIN_DIR
 WORKDIR $FIN_DIR
 # Can be a branch or a SHA
-ARG FIN_BRANCH=260d9465d5ee71cc8dc7715ba583107ac4556f93
+ARG FIN_BRANCH=04508bcfe479310faafa4de8c7fe92e8cdf7ce5f
```

Comment:
If you could link this off the develop branch when the linked Fin PR clears, that would be great.

Reply:
Updated in develop!

Reply:
Not sure what you mean @JehandadKhan, my Tuna branch is pointing to my Fin branch, which is up to date with develop https://github.com/ROCmSoftwarePlatform/fin/pull/64